lib/consyncful/item_mapper.rb

```diff
@@ -35,6 +39,23 @@ def mapped_fields(default_locale)
 
     private
 
+    # rubocop:disable Metrics/AbcSize
+    def tagged_content?(item)
```

Comment:
This method name isn't quite right. Maybe with my request above, there is a way to make that a bit clearer (move this logic into `excluded_by_tag?` or something.

I also think you should try to solve the complexity limit here for ABC size. Potentially separate the tag extraction step & guard cluses from assessing whether the tags match.